src/java.base/share/classes/jdk/internal/javac/PreviewFeature.java

```diff
@@ -81,6 +81,8 @@ public enum Feature {
         CLASSFILE_API,
         @JEP(number=461, title="Stream Gatherers", status="Preview")
         STREAM_GATHERERS,
+        @JEP(number=0, title="Module Imports", status="Preview")
```

Comment:
I see this has been assigned JEP 476 so I assume you can set the number for the first integration.

Reply:
Done, thanks:
https://github.com/openjdk/jdk/pull/18614/commits/432393abb4ac1f5c4730d982a3911284fe866318